include/grpcpp/support/proto_buffer_reader.h

```diff
@@ -148,7 +148,7 @@ class ProtoBufferReader : public grpc::protobuf::io::ZeroCopyInputStream {
             *slice(), GRPC_SLICE_LENGTH(*slice()) - backup_count(),
             GRPC_SLICE_LENGTH(*slice()) - backup_count() + count)));
       }
-      int64_t take = std::min(backup_count(), static_cast<int64_t>(count));
+      int64_t take = (std::min)(backup_count(), static_cast<int64_t>(count));
```

Comment:
why are there brackets around std::min?

Reply:
brackets around std::min avoid issues where an older header that #define's a min 'function'

Reply:
Looks like mergeable has been running for a couple of days now. Does it need restarting? Or do I need to rebase once more?